src/dialogs/more-info/components/cover/ha-more-info-cover-tilt-position.ts

```diff
@@ -0,0 +1,128 @@
+import {
+  css,
+  CSSResultGroup,
+  html,
+  LitElement,
+  TemplateResult,
+  unsafeCSS,
+} from "lit";
+import { customElement, property, state } from "lit/decorators";
+import { styleMap } from "lit/directives/style-map";
+import { computeAttributeNameDisplay } from "../../../../common/entity/compute_attribute_display";
+import { stateColorCss } from "../../../../common/entity/state_color";
+import "../../../../components/ha-control-slider";
+import { CoverEntity } from "../../../../data/cover";
+import { UNAVAILABLE } from "../../../../data/entity";
+import { HomeAssistant } from "../../../../types";
+
+function createTiltSliderTrackBackgroundGradient(
+  count = 24,
+  minStrokeWidth = 0.2
+) {
+  const gradient: [number, string][] = [];
+
+  for (let i = 0; i < count; i++) {
+    const stopOffset1 = i / count;
+    const stopOffset2 =
+      stopOffset1 +
+      (i / count ** 2) * (1 - minStrokeWidth) +
+      minStrokeWidth / count;
+
+    if (i !== 0) {
+      gradient.push([stopOffset1, "transparent"]);
+    }
+    gradient.push([stopOffset1, "var(--control-slider-color)"]);
+    gradient.push([stopOffset2, "var(--control-slider-color)"]);
+    gradient.push([stopOffset2, "transparent"]);
+  }
+
+  return unsafeCSS(
+    gradient
+      .map(([stop, color]) => `${color} ${(stop as number) * 100}%`)
+      .join(", ")
+  );
+}
+
+const GRADIENT = createTiltSliderTrackBackgroundGradient();
```

Comment:
Since we dont do this dynamic anyway (we dont pass any params) should we not just hardcode the css?

Reply:
The gradient looks like this 😅. I think it's better to generate it with a function. I removed the parameter as there was not used.
```css
--control-slider-background: -webkit-linear-gradient(top, var(--control-slider-color) 0%, var(--control-slider-color) 0.8333333333333334%, transparent 0.8333333333333334%, transparent 4.166666666666666%, var(--control-slider-color) 4.166666666666666%, var(--control-slider-color) 5.138888888888888%, transparent 5.138888888888888%, transparent 8.333333333333332%, var(--control-slider-color) 8.333333333333332%, var(--control-slider-color) 9.444444444444445%, transparent 9.444444444444445%, transparent 12.5%, var(--control-slider-color) 12.5%, var(--control-slider-color) 13.750000000000002%, transparent 13.750000000000002%, transparent 16.666666666666664%, var(--control-slider-color) 16.666666666666664%, var(--control-slider-color) 18.055555555555554%, transparent 18.055555555555554%, transparent 20.833333333333336%, var(--control-slider-color) 20.833333333333336%, var(--control-slider-color) 22.36111111111111%, transparent 22.36111111111111%, transparent 25%, var(--control-slider-color) 25%, var(--control-slider-color) 26.66666666666667%, transparent 26.66666666666667%, transparent 29.166666666666668%, var(--control-slider-color) 29.166666666666668%, var(--control-slider-color) 30.97222222222223%, transparent 30.97222222222223%, transparent 33.33333333333333%, var(--control-slider-color) 33.33333333333333%, var(--control-slider-color) 35.27777777777778%, transparent 35.27777777777778%, transparent 37.5%, var(--control-slider-color) 37.5%, var(--control-slider-color) 39.583333333333336%, transparent 39.583333333333336%, transparent 41.66666666666667%, var(--control-slider-color) 41.66666666666667%, var(--control-slider-color) 43.88888888888889%, transparent 43.88888888888889%, transparent 45.83333333333333%, var(--control-slider-color) 45.83333333333333%, var(--control-slider-color) 48.19444444444444%, transparent 48.19444444444444%, transparent 50%, var(--control-slider-color) 50%, var(--control-slider-color) 52.5%, transparent 52.5%, transparent 54.166666666666664%, var(--control-slider-color) 54.166666666666664%, var(--control-slider-color) 56.80555555555556%, transparent 56.80555555555556%, transparent 58.333333333333336%, var(--control-slider-color) 58.333333333333336%, var(--control-slider-color) 61.111111111111114%, transparent 61.111111111111114%, transparent 62.5%, var(--control-slider-color) 62.5%, var(--control-slider-color) 65.41666666666667%, transparent 65.41666666666667%, transparent 66.66666666666666%, var(--control-slider-color) 66.66666666666666%, var(--control-slider-color) 69.72222222222221%, transparent 69.72222222222221%, transparent 70.83333333333334%, var(--control-slider-color) 70.83333333333334%, var(--control-slider-color) 74.02777777777779%, transparent 74.02777777777779%, transparent 75%, var(--control-slider-color) 75%, var(--control-slider-color) 78.33333333333333%, transparent 78.33333333333333%, transparent 79.16666666666666%, var(--control-slider-color) 79.16666666666666%, var(--control-slider-color) 82.63888888888889%, transparent 82.63888888888889%, transparent 83.33333333333334%, var(--control-slider-color) 83.33333333333334%, var(--control-slider-color) 86.94444444444444%, transparent 86.94444444444444%, transparent 87.5%, var(--control-slider-color) 87.5%, var(--control-slider-color) 91.25%, transparent 91.25%, transparent 91.66666666666666%, var(--control-slider-color) 91.66666666666666%, var(--control-slider-color) 95.55555555555554%, transparent 95.55555555555554%, transparent 95.83333333333334%, var(--control-slider-color) 95.83333333333334%, var(--control-slider-color) 99.86111111111111%, transparent 99.86111111111111%);
```